Formula/sourcedocs.rb

```diff
@@ -0,0 +1,21 @@
+class Sourcedocs < Formula
+  desc "Generate Markdown files from inline source code documentation"
+  homepage "https://github.com/eneko/SourceDocs"
+  url "https://github.com/eneko/SourceDocs/archive/0.5.1.tar.gz"
+  sha256 "3c2e2de695d49dbdd5acb49f8876042bdc97e8d6b95584d3ef6b592b8f10affc"
+  head "https://github.com/eneko/SourceDocs.git"
+
+  depends_on :xcode
```

Comment:
Since different major versions of Swift are not source-level compatible, we may want to explicitly specify the version of Xcode, e.g. Xcode 10 if Swift 4.2 is required.